### ISSUE TITLE
Interferes with Reddit comment submission

### ISSUE THREAD

**Issue by huachuman** [ynshung/better-yt-shorts#200]
**Describe the bug**
 I have been experiencing an issue where trying to submit a reddit comment (e.g. by pressing control+enter or by pressing the reply button) does nothing. Disabling your extension fixes the issue.

**To Reproduce**
1. Go to reddit and try to submit a comment.

**Expected behavior**
The comment submits.

**Screenshots**
NA

**Desktop (please complete the following information):**

- OS: Windows 11
- Vivaldi
- 6.5.3206.48 (Stable channel)

**Additional context**
NA

**Comment by samisul** [comment]
Now that's a VERY weird issue, especially since the content script should be injected anywhere other than youtube.

**Comment by ynshung** [comment]
Agree, could it be a specific issue with the Vivaldi browser? Maybe this issue can be tested in that particular browser.

**Comment by huachuman** [comment]
I think we should close this for now. Here's why:

First of all, here is what is absolutely true:
- For the last month or so I have not been able to press the reply button on reddit (reason I started investigating extensions)
- The reply button works perfectly fine in other browsers so I can confirm 100% it's not related to my reddit account.
- As soon as I disabled this extension and reloaded the page, I was able to press the reply button. HOWEVER
- This appeared to be an isolated incident because I can no longer press the reply button, even as the extension is disabled.

I'm wondering if something is happening with javascript that is interfering with the page, and that maybe disabling this extension unstuck that for a short period, but now it's stuck again.

I agree this was very weird, and it probably doesn't have anything to do with your extension.
